Sources/Datadog/RUM/AutoInstrumentation/RUMAutoInstrumentation.swift

```diff
@@ -70,8 +69,9 @@ internal class RUMAutoInstrumentation {
                 userActions = nil
             }
         } catch {
-            userLogger.warn("🔥 RUM automatic tracking can't be set up due to error: \(error)")
-            developerLogger?.warn("🔥 RUM automatic tracking can't be set up due to error: \(error)")
+            consolePrint(
+                "🔥 Datadog SDK error: RUM automatic tracking can't be set up due to error: \(error)"
+            )
```

Comment:
It's not a change of the trouble-reporting approach 🙂 - this one was just incorrect. The swizzling is applied at the SDK initialization time, when the usage of `userLogger` is invalid, because this logger is flagged by the `Datadog.verbosity = .debug` setup applied by the user after the SDK is initialised. In many other places, `user` / `dev` loggers are used to report problems **after** SDK initialization, which is correct.